litr-demo/src/main/java/com/linkedin/android/litr/demo/data/RecordCameraPresenter.kt

```diff
@@ -31,6 +32,7 @@ class RecordCameraPresenter(
     private val mediaTransformer: MediaTransformer
 ) : TransformationPresenter(context, mediaTransformer) {
 
+    @SuppressLint("MissingPermission")
```

Comment:
Since this is demo app code, I thought I would just suppress the warning rather than doing all the plumbing to expose the required permission. In prod code, the MediaSource correctly exposes this and the consumer can handle it accordingly.